### DISCUSSION TITLE
Azure AD SAML not working

### DISCUSSION THREAD

**Discussion by hyouhaku** [kimai/kimai#3841] [Questions & Answers]
hi all, just installed version 2.0.0 from `kimai/kimai2:apache-2.0.0-prod` docker image, everything is working beautifully except for SAML 
I've configured local.yaml and placed it under /opt/kimai/config/packaged, reloaded cache, however it's still showing same password login screen
anything wrong that I did?

**Comment by kevinpapst** [comment]
It works when you place it at `config/packages/local.yaml`.

Also check your system configuration Authentication `yourURL/en/admin/system-config/#conf_authentication`

**Comment by hyouhaku** [comment]
thanks @kevinpapst , so local.yaml is placed under /opt/kimai/config/packages/
cache is reloaded successfully
but under authentication configuration section i can only see options for "forgot password"

**Reply by kevinpapst** [reply]
What happens if you deliberately destroy the yaml structure? 
Just type any garbage into the local.yaml file => do you see an error upon cache reload?

I can tell you that SAML works. So it must be a docker/environment issue.
I do not use Docker and you might want to post in the docker repository, because I can't really help with it.

**Reply by hyouhaku** [reply]
just to let you know that changing local.yaml file does cause error message upon cache reload, sure I'll check with docker side

**Comment by hyouhaku** [comment]
in case anyone gets stuck with same issue in the future, I figured out that in my case local.yaml wasn't really propagated/loaded even with cache reloading. 
config file has to exist before all the initialization to be loaded properly.

**Comment by hyouhaku** [comment]
@kevinpapst any idea why kimai:reset:locales was called when first time a user signed in through Azure AD? environment setting is prod, so I guess that's not supposed to happen?
